UnityProject/Assets/Scripts/UI/Systems/AdminTools/DevTools/GUI_DevCloner.cs

```diff
@@ -49,6 +49,34 @@ void Awake()
 		ToState(State.SELECTING);
 	}
 
+	private void CheckAndApplyPalette(ref SpriteRenderer renderer)
+	{
+		bool isPaletted = false;
+
+		if (toClone.TryGetComponent(out ItemAttributesV2 prefabAttributes))
+		{
+			ItemsSprites sprites = prefabAttributes.ItemSprites;
+			if (sprites.IsPaletted)
+			{
+				List<Color> palette = sprites.Palette;
+				Debug.Assert(palette != null, "Palette must not be null on paletteable objects.");
+				MaterialPropertyBlock block = new MaterialPropertyBlock();
+				renderer.GetPropertyBlock(block);
+				List<Vector4> pal = palette.ConvertAll((Color c) => new Vector4(c.r, c.g, c.b, c.a));
```

Comment:
Codacy found an issue: [Remove the type specification; it is redundant.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6381514)